src/types/websockets/ws-api-requests.ts

```diff
@@ -499,20 +495,20 @@ export interface WSAPIFuturesTickerBookRequest {
  * Futures trading request types
  */
 
-export interface WSAPINewFuturesOrderRequest<numberType = number> {
+export interface WSAPINewFuturesOrderRequest {
```

Comment:
```suggestion
export interface WSAPINewFuturesOrderRequest<numberType = numberInString> {
```

Can you restore the generic? Just change the default assignment to `numberInString` (instead of the = number that it was before)